cyvcf2/cyvcf2.pxd

```diff
@@ -78,7 +81,7 @@ cdef extern from "htslib/hts.h":
 cdef extern from "htslib/tbx.h":
 
     ctypedef struct tbx_t:
-        pass
```

Comment:
I added this so that Cython "knows about" the idx member of this struct. However, it has two other members - I'm not sure if it's safe to just ignore them? Originally I copied the full struct definition in here, but that seems worse, if it's not needed.

Reply:
I tried to find documentation of this in cython, but failed. My memory is that you should either declare none of the fields or all of the fields. The definition of `tbx_t` is simple enough that it is at least easy explicitly list all of them.

Reply:
Fixed